guides/prereq/gateway-provider/istio.helmfile.yaml

```
@@ -19,7 +19,7 @@ releases:
       - meshConfig:
           defaultConfig:
             proxyMetadata:
-              SUPPORT_GATEWAY_API_INFERENCE_EXTENSION: true
+              SUPPORT_GATEWAY_API_INFERENCE_EXTENSION: "true"
         pilot:
           env:
             SUPPORT_GATEWAY_API_INFERENCE_EXTENSION: true
```

After looking at this I believe the same is true with pilot env values:
```suggestion
            SUPPORT_GATEWAY_API_INFERENCE_EXTENSION: "true"
```